packages/asc-ui/src/components/CompactPager/CompactPager.test.tsx

```diff
@@ -0,0 +1,102 @@
+import * as React from 'react'
+import { render, fireEvent } from '@testing-library/react'
+import { ThemeProvider, ascDefaultTheme } from '@datapunt/asc-core'
+import CompactPager, { determineTotalPages } from './CompactPager'
+
+const PREV_BUTTON = 'prevbutton'
+const NEXT_BUTTON = 'nextbutton'
+
+describe('CompactPager', () => {
+  it('should change the page when pressing prev button', () => {
+    const onChangePageMock = jest.fn()
+
+    const { getByTestId } = render(
+      <ThemeProvider theme={ascDefaultTheme}>
+        <CompactPager
+          page={2}
+          pageSize={20}
+          collectionSize={60}
+          onPageChange={onChangePageMock}
+        />
+      </ThemeProvider>,
+    )
+
+    fireEvent.click(getByTestId(PREV_BUTTON))
+    expect(onChangePageMock).toHaveBeenCalledWith(1)
+  })
+
+  it('should not change the page at the bounds when pressing the prev button', () => {
+    const onChangePageMock = jest.fn()
+
+    const { getByTestId } = render(
```

Comment:
Perhaps create a createComponent function with props as arguments to make tests a bit more compact / less repetitive?

Reply:
Yes, I pushed a change that should resolve this.